utils/imagenet_topk_accuracy_driver.py

```diff
@@ -0,0 +1,318 @@
+        # classify (if applicable) and the log of image-classifer output.
+        tmp_dir_name = tempfile.mkdtemp()
+        path_to_tmp_log = os.path.join(tmp_dir_name, "log.txt")
+        fout = file(path_to_tmp_log,'w')
+        p.logfile = fout
+
+        if args.verbose:
+            print "Temp log located at: " + path_to_tmp_log
+
+        prompt = 'Enter image filenames to classify: '
+        top1_count = 0
+        top5_count = 0
+        total_count = 0
+
+        # Process the images in batches as specified on the command line.
+        for img_index in xrange(0, total_image_count, args.batch_size):
+            curr_img_paths = get_curr_img_paths(img_paths, img_index,
+                                                tmp_dir_name, args.batch_size,
+                                                args.resize_input_images)
+
+            # Expect prompt from the image-classifier for the next image path.
+            p.expect(prompt)
+
+            appended_paths = " ".join(curr_img_paths)
+            assert len(appended_paths) <= 1024, "Single line length is too long for terminal: %r" % len(appended_paths)
+
+            # Send the paths to the image-classifier.
+            p.sendline(appended_paths)
+
+            for batch_idx in xrange(args.batch_size):
+                # Now we expect the image-classifier's response with the label.
+                # The first line will include the path to the file, e.g.:
+                #  File: tests/images/imagenet/cat_285.png
+                p.expect(" File: " + curr_img_paths[batch_idx])
+
+                # All labels will be formatted like:
+                # Label-K1: 281 (probability: 0.7190)
+                top5_labels = []
+                for x in xrange(5):
+                    label_and_prob = p.readline()
+                    # Get the label from the line.
+                    label = label_and_prob.split()[1]
+                    top5_labels.append(int(label))
+
+                expected_label = img_labels[img_index + batch_idx]
+                if expected_label == top5_labels[0]:
+                    top1_count = top1_count + 1
+                if expected_label in top5_labels:
+                    top5_count = top5_count + 1
+
+            curr_completed_count = img_index + args.batch_size
+            if curr_completed_count % 100 == 0:
+                print("Finished image index " + str(curr_completed_count) +
+                      " out of " + str(total_image_count)),
+                if args.verbose:
+                      print("; current Top-1/5 accuracy:")
+                      print_topk_accuracy(curr_completed_count, top1_count, top5_count)
+                else:
+                      print("")
+
+    finally:
+        p.close(force=True)
+
+        # Remove the temp directory we used to save the images and log.
+        if tmp_dir_name:
+            shutil.rmtree(tmp_dir_name)
+
+    print("\nCompleted running; Final Top-1/5 accuracy across " +
```

Comment:
I see a mix of double-quotes and single quotes for strings. And also `print` sometimes with brackets and sometimes without. I believe pylint can detect this too.